prime-router/src/testIntegration/kotlin/datatests/mappinginventory/catchall/orcobr/ORCOBRToServiceRequestDiagnosticReportTests.kt

```diff
@@ -72,4 +72,22 @@ class ORCOBRToServiceRequestDiagnosticReportTests {
         val testFileName = "catchall/orcobr/orc_obr-to-servicerequest_diagnosticreport-obr8-not-populated"
         assert(verifyHL7ToFHIRToHL7Mapping(testFileName).passed)
     }
+
```

Comment:
I welcome feedback if anyone thinks I'm over-testing/testing in the wrong place.

I wanted to capture the following scenarios:
- HL7 -> FHIR -> HL7 when OBR.25 is a value that is fully mapped. 
  - OBR.25 = F; DR.Status is set 'final'; OBRExtension preserves the "F"
- HL7 -> FHIR -> HL7 when OBR.25 is a value that does not have a FHIR mapping. 
  - OBR.25 = Z; DR.Status is set 'unknown'; OBRExtension preserves the "Z"
- FHIR -> HL7 when DR.Status is present but with an unmapped value and 
  - DR.Status is 'entered-in-error'; OBRExtension not populated; Final OBR.25 is not present.